docs/i18n.rst

```diff
@@ -258,7 +257,7 @@ This command will extract front- and backend strings and upload them to Crowdin,
 
 .. code-block:: bash
 
-  make i18n-upload branch=[release-branch-name]
+  make i18n upload-sources --branch=[release-branch-name]
```

Comment:
I don't see any changes to the Makefile in this PR, so not sure how this command name has been updated, and even if it had, don't think it would have a space in it.